Sources/Build/Command.compile(ClangModule).swift

```diff
@@ -105,6 +105,7 @@ extension Command {
         var args = module.basicArgs
         args += module.optimizationFlags(conf)
         args += ["-L\(prefix)"]
+        args += module.languageArgs
```

Comment:
I think the name is bad then ... should I rename to lanugageLinkArgs?
(Note that this is linking stage)